### PR TITLE
VACMS-10936: Reusable Q&A Runbook

### LINKED ISSUE #10936: BE: Document the process to expose Q&A on new content types
## Description
We want to eventually add Q&As to as many content types as possible / that make sense. We will develop the steps to achieve that as an instruction manual to ourselves or future Drupal developers (runbook), and document them for posterity.

## Eng notes
We've done this before in https://github.com/department-of-veterans-affairs/va.gov-cms/issues/9737, can use that as a reference.

## Tasks
* Add Q&A with Place Q&A modal to a new content type, similar to the R&S & FAQ implementation
* Document the steps as a runbook that can be used for future content types
* Runbook can either be a Github document in the [COPE docs in VA.gov Team repo Products repo](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/content/tier-2-content-IA-and-design/cope-cms-faq-library/product-outline.md), OR an issue template

Since this won't be a repetitive task forever, just until we get through all content types, can be a Github doc rather than an issue template.

## Acceptance Criteria

- [ ] PO expects that runbook author has actually performed the steps and verified them before/while documenting
- [ ] PO expects that Drupal steps have been added to the runbook here: https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/content/tier-2-content-IA-and-design/cope-cms-faq-library/engineering 
- [x] PO expects that the runbook addresses Drupal steps only. FE steps will be covered via #10937

### PR DESCRIPTION
**Pull request by chri5tia**
Adds documentation for the "COPE CMS FAQ Library" e.g. the Reusable Q&As implementation.

Closes [#10936 in va.gov-cms.](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/10936).

### CODE REVIEW

**Review by jilladams — CHANGES_REQUESTED** [review]
@chri5tia happy to see this headed out. However: changes need to move into a different doc: https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/content/tier-2-content-IA-and-design/cope-cms-faq-library/engineering, parallel to this one.

**Review by jilladams — APPROVED** [review]
@chri5tia approving the fact that it moved to a new doc. I don't work in config anymore, so I'm adding Daniel as a reviewer and his approval should be what unblocks merge, if the notes here make sense to him, rather than merging on my approval.

File: products/content/tier-2-content-IA-and-design/cope-cms-faq-library/engineering/README.md [hunk 1]
```diff
@@ -1,5 +1,33 @@
 # Exposing Q&A on New Content Types
 
+## CMS Instructions: Reusable Q&A Implementation Runbook
+For existing content types, edit the field_content_block field (Main content) if it exists on the content type, select to add the Q&A paragraph type, then save.
+
+Now when editors want to add an existing reusable Q&A they will see it as one of the options for adding main content.
+
+1. In the Drupal UI > Structure > Content types and select your content type.
+2. Edit the chosen Entity Reference Revisions field or add a new field, type Paragraph under Reference Revisions.
+3. If editing existing field, skip the following step. If it's a new field, select Paragraph for "Type of item to reference" and select Unlimited cardinality/allowed number of values.
```
Comment (dsasser) [inline products/content/tier-2-content-IA-and-design/cope-cms-faq-library/engineering/README.md:10]: CONSIDER:
The if/then scenarios could cause some confusion. Consider breaking the steps into distinct groups: creating a new field, and utilizing an existing field. There might be some redundancies, but I think it is worth that for extra clarity/ease of following.

File: products/content/tier-2-content-IA-and-design/cope-cms-faq-library/engineering/README.md [hunk 2]
```diff
@@ -1,5 +1,33 @@
 # Exposing Q&A on New Content Types
 
+## CMS Instructions: Reusable Q&A Implementation Runbook
+For existing content types, edit the field_content_block field (Main content) if it exists on the content type, select to add the Q&A paragraph type, then save.
+
+Now when editors want to add an existing reusable Q&A they will see it as one of the options for adding main content.
+
+1. In the Drupal UI > Structure > Content types and select your content type.
```
Comment (dsasser) [inline products/content/tier-2-content-IA-and-design/cope-cms-faq-library/engineering/README.md:8]: SHOULD:

> select your content type.

'Manage fields' is the link that should be clicked. There is no link for a content type on this page.

**Review by dsasser — CHANGES_REQUESTED** [review]

File: products/content/tier-2-content-IA-and-design/cope-cms-faq-library/engineering/README.md (hunk 2 above)
Comment (chri5tia) [reply products/content/tier-2-content-IA-and-design/cope-cms-faq-library/engineering/README.md]: Changed to "In the Drupal UI > Structure > Content types, click "Manage fields" for your content type."

**Review by dsasser — APPROVED** [review]
Looks good!

**Pull request merged**